nih-sparc/sparc-app
# Find Data - Dataset ordering, Remove filters #4m8ejt

# Description

The purpose of this PR is to order datasets/simulations results in Find Data by their date, unless the user has typed in a query. In addition, filters functionality has been removed.

## Type of change

# How Has This Been Tested?

- Go to Find Data
- The filters button should no longer be there
- The datasets should be ordered by date, starting with the most recently updated
- Type in a search query
- The datasets should be ordered by relevance. To confirm this, open up the network tab and look at the endpoint request. The `orderBy` param should not be in this request.
- Go to the Simulations tab
- The datasets should be ordered by date, starting with the most recently updated
- Type in a search query
- The datasets should be ordered by relevance. To confirm this, open up the network tab and look at the endpoint request. The `orderBy` param should not be in this request.

# Checklist:
